src/System.Collections.Concurrent/tests/ConcurrentQueueTests.cs

```diff
@@ -17,120 +17,7 @@ public class ConcurrentQueueTests : ProducerConsumerCollectionTests
         protected override bool IsEmpty(IProducerConsumerCollection<int> pcc) => ((ConcurrentQueue<int>)pcc).IsEmpty;
         protected override bool TryPeek(IProducerConsumerCollection<int> pcc, out int result) => ((ConcurrentQueue<int>)pcc).TryPeek(out result);
         protected override bool ResetImplemented => false;
-
-        [Fact]
-        public void Ctor_NoArg_ItemsAndCountMatch()
```

Comment:
I do not see changes like
```cs
-
-        [Fact]
-        public void Ctor_NoArg_ItemsAndCountMatch()
```

I see next changes
```
-        protected override bool IEnumerable_Generic_Enumerator_Current_EnumerationNotStarted_ThrowsInvalidOperationException => false;
```

I removed this flag because it not used more. But I did not remove any test like `public void Ctor_NoArg_ItemsAndCountMatch()`